symengine/lib/symengine_wrapper.pyx

```diff
@@ -4801,10 +4800,80 @@ IF HAVE_SYMENGINE_LLVM:
+            """
+            cdef bytes s = self.lambda_double[0].dumps()
+            return llvm_float_loading_func, (self.args_size, self.tot_out_size, self.out_shapes, self.real, \
+                self.n_exprs, self.order, self.accum_out_sizes, self.numpy_dtype, s)
+
+        cpdef unsafe_real(self, float[::1] inp, float[::1] out, int inp_offset=0, int out_offset=0):
+            self.lambda_double[0].call(&out[out_offset], &inp[inp_offset])
+
+        cpdef unsafe_eval(self, inp, out, unsigned nbroadcast=1):
+            cdef float[::1] c_inp, c_out
+            cdef unsigned idx
+            c_inp = np.ascontiguousarray(inp.ravel(order=self.order))
+            c_out = out
+            for idx in range(nbroadcast):
+                self.lambda_double[0].call(&c_out[idx*self.tot_out_size], &c_inp[idx*self.args_size])
+
+    IF HAVE_SYMENGINE_LLVM_LONG_DOUBLE:
+        cdef class LLVMLongDouble(_LLVMLambdify):
+            def __cinit__(self, args, *exprs, cppbool real=True, order='C', cppbool cse=False, cppbool _load=False, opt_level=3):
+                self.opt_level = opt_level
+
+            cdef _init(self, symengine.vec_basic& args_, symengine.vec_basic& outs_, cppbool cse):
+                self.lambda_double.resize(1)
+                self.lambda_double[0].init(args_, outs_, cse, self.opt_level)
+
+            cdef _load(self, const string &s):
+                self.lambda_double.resize(1)
+                self.lambda_double[0].loads(s)
+
+            def __reduce__(self):
+                """
+                Interface for pickle. Note that the resulting object is platform dependent.
+                """
+                cdef bytes s = self.lambda_double[0].dumps()
+                return llvm_long_double_loading_func, (self.args_size, self.tot_out_size, self.out_shapes, self.real, \
+                    self.n_exprs, self.order, self.accum_out_sizes, self.numpy_dtype, s)
+
+            cpdef unsafe_real(self, long double[::1] inp, long double[::1] out, int inp_offset=0, int out_offset=0):
+                self.lambda_double[0].call(&out[out_offset], &inp[inp_offset])
+
+            cpdef unsafe_eval(self, inp, out, unsigned nbroadcast=1):
+                cdef long double[::1] c_inp, c_out
+                cdef unsigned idx
+                c_inp = np.ascontiguousarray(inp.ravel(order=self.order))
+                c_out = out
+                for idx in range(nbroadcast):
+                    self.lambda_double[0].call(&c_out[idx*self.tot_out_size], &c_inp[idx*self.args_size])
+
     def llvm_loading_func(*args):
         return LLVMDouble(args, _load=True)
 
-def Lambdify(args, *exprs, cppbool real=True, backend=None, order='C', as_scipy=False, cse=False, **kwargs):
+    def llvm_float_loading_func(*args):
+        return LLVMFloat(args, _load=True)
+
+    def llvm_long_double_loading_func(*args):
+        return LLVMLongDouble(args, _load=True)
```

Comment:
Looks like this line needs to be conditionally included? using
```
   IF HAVE_SYMENGINE_LLVM_LONG_DOUBLE:
```
cf. https://ci.appveyor.com/project/isuruf/symengine-py-l1jmr/builds/30980461/job/57ju8pg99uquiiwc#L800